slow-restart/README.md

```diff
@@ -0,0 +1,29 @@
+# slow-restart
+
+Script slowly restarts all deployments in `default` namespace for which linkerd-proxy version differs with
+linkerd-proxy-injector from Control Plane.
```

Comment:
```suggestion
```suggestion
Script restarts all deployments one-by-one from `default` namespace for which `linkerd-proxy` version differs with
`linkerd-proxy-injector` from Control Plane.
```